packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -204,6 +219,29 @@ class SnackBar extends StatefulWidget {
   /// used, if that is also null, the default value is 6.0.
   final double elevation;
 
+  /// Empty space to surround the snack bar.
+  ///
+  /// This property is only used when [behavior] is [SnackBarBehavior.floating].
+  ///
+  /// If this property is null, then the default is
+  /// `EdgeInsets.fromLTRB(15.0, 5.0, 15.0, 10.0)`.
+  final EdgeInsetsGeometry margin;
+
+  /// The amount of horizontal space to apply to the snack bar's content and
+  /// optional action.
+  ///
+  /// If this property is null, then the default depends on the [behavior] and
+  /// the presence of an [action].
```

Comment:
This could be more specific and mention that:
For a floating snackbar the start padding is 16, and for non-floating 24. If it includes an action, the same padding will be added at the end.

Reply:
done